### PR TITLE
Refactor custom css file split into multiple files

### PR DESCRIPTION
**Pull request by neomatrix369**
- Extracted global styles, layout, navbar/navigation, buttons, colours and media queries from the custom.css into their own files (custom-xxxx-xxxxx.css) and added them to the head.html to make them available to all other pages
- Added CSS specificity to the media queries and a link to the resource on how to calculate it
- Reordered a few selectors in the order of specificity - smallest to highest (more generic to more specific)
- Renamed custom.css to custom-layout.css and also changed it in the head.html to make it available to all other pages
- Amended scripts and config files related to building the Jekyll site on the localmachine

Related to issue #529

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -7,12 +7,8 @@ RSpec::Core::RakeTask.new(:spec) do |task|
   task.rspec_opts = '--color'
 end
 
-task :build do
-  sh 'bundle exec jekyll build --config _config.yml,_config_en.yml --trace'
-end
-
 task :buildboth do
-  sh 'bundle exec jekyll build --trace'
+  sh 'bundle exec jekyll build --config _config.yml,_config_en.yml --trace'
```
Comment (jukecraft) [inline Rakefile]: why don't we want to have a default build option?

File: assets/css/custom/custom-layout-buttons.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    background-color: transparent;
+    border: 1px solid white;
+    color: white;
+}
+
+.btn-u-orange {   /* 0 0 1 0 */
+    background: #e67e22;
+}
+
+#sharethisButtons {   /* 0 0 1 0 */
```
Comment (jukecraft) [inline assets/css/custom/custom-layout-buttons.css]: why is this one not using hyphens?

File: assets/css/custom/custom-layout-buttons.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+.btn-brd.btn-u-orange:hover {   /* 0 0 3 0 */
+    color: #d35400;
+    border-color: #d35400;
+}
+
+.btn-brd.btn-u-orange.btn-brd-hover:hover {   /* 0 0 4 0 */
+    background: #d35400;
+}
+
+#sharethisButtons {   /* 0 1 0 0 */
```
Comment (jukecraft) [inline assets/css/custom/custom-layout-buttons.css:80]: duplicate from above?

File: assets/css/custom/custom-layout-colours.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+.white,
+p.white,
+ul.white li {
+    color: white;
+}
+
+/* Section colours */
+
+.gray-section {   /* 0 0 1 0 */
```
Comment (jukecraft) [inline assets/css/custom/custom-layout-colours.css]: duplicate?

File: assets/css/custom/custom-navbar-or-navigation.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+   Navbar or Navigation
+   -------------------------------------- */
+
+/*
+   CSS specificity calculator: https://specificity.keegan.st/
+   Specificity components: [Inline styles] [IDs] [Classes, attributes and pseudo-classes] [Elements and pseudo-elements]
+   Specificity ranges from lowest to highest (more generic to more specific)
+ */
+
+.breadcrumbs h2 {  /* 0 0 1 1 */
```
Comment (jukecraft) [inline assets/css/custom/custom-navbar-or-navigation.css]: is this part of the navigation?

File: buildAndRunSite.sh [hunk 6]
```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-jekyll serve --incremental --watch
\ No newline at end of file
+jekyll serve --incremental --watch --config _config.yml,_config_en.yml
```
Comment (jukecraft) [inline buildAndRunSite.sh:3]: maybe change the README as well?

File: Rakefile (hunk 1 above)
Comment (neomatrix369) [reply Rakefile]: It's doable, and we intend to do it cleaner in another PR, but that would mean changes to the config files (both Rake and _config files). 

This change here was to remove tasks that didn't work or was redundant.

File: buildAndRunSite.sh (hunk 6 above)
Comment (neomatrix369) [reply buildAndRunSite.sh:3]: The readme refers to running the buildAndRunSite.sh, but we will be changing the way we build the two sites, so not yet.

File: assets/css/custom/custom-layout-buttons.css (hunk 3 above)
Comment (neomatrix369) [reply assets/css/custom/custom-layout-buttons.css:80]: indeed, i;'ll remove it

File: assets/css/custom/custom-navbar-or-navigation.css (hunk 5 above)
Comment (neomatrix369) [reply assets/css/custom/custom-navbar-or-navigation.css]: Its not used, i'll remove it - thnx

**Comment by neomatrix369** [comment]
Checked the relevant pages, no regression found after applying these changes.

File: Rakefile (hunk 1 above)
Comment (jukecraft) [reply Rakefile]: For now, can we not keep the build task even if it's a copy?

**Comment by jukecraft** [comment]
@neomatrix369 can you take care of the conflicts please?

File: assets/css/custom/custom-layout-buttons.css (hunk 3 above)
Comment (jukecraft) [reply assets/css/custom/custom-layout-buttons.css:80]: I think you  removed the other one, the name still stands out

**Comment by neomatrix369** [comment]
@franziskas just resolved the conflicts

File: Rakefile (hunk 1 above)
Comment (neomatrix369) [reply Rakefile]: the old build task won't work without the added parameters, which makes one of them redundant

File: assets/css/custom/custom-layout-buttons.css (hunk 2 above)
Comment (neomatrix369) [reply assets/css/custom/custom-layout-buttons.css]: Not changing names of selectors via this PR we could do this via another one

File: assets/css/custom/custom-layout-buttons.css (hunk 3 above)
Comment (neomatrix369) [reply assets/css/custom/custom-layout-buttons.css:80]: As mentioned earlier not changing the name of this selector, in this PR, maybe via another PR

File: assets/css/custom/custom-layout-colours.css (hunk 4 above)
Comment (neomatrix369) [reply assets/css/custom/custom-layout-colours.css]: Has been removed

File: Rakefile (hunk 1 above)
Comment (jukecraft) [reply Rakefile]: Exactly. Why not keep the one from the readme people are used to?  You're breaking the interface

Comment (neomatrix369) [reply Rakefile]: Just added a commit to address this so the README instructions stay good

**Pull request merged**